rpc/smoldot.ts

```diff
@@ -8,23 +8,29 @@ import { RpcEgressMessage } from "./rpc_messages.ts"
 export interface SmoldotRpcConnProps {
   relayChainSpec: string
   parachainSpec?: string
+  maxLogLevel?: number
 }
 
 let client: undefined | Client
+let count = 0
 
 export class SmoldotConnection extends Connection {
   // private so that the types don't need to be referenced when packaged
   private smoldotChainPending
   listening
   stopListening
+  chainsCount
 
   constructor(readonly props: SmoldotRpcConnProps) {
     super()
+    this.chainsCount = props.parachainSpec ? 2 : 1
```

Comment:
If this is only ever used to increment/decrement `count`, I believe it can always just be 1.